docs/_sections/capture.md

```diff
@@ -3,4 +3,107 @@ layout: post
 title: Capture
 ---
 
+## DVRescue GUI
+Step-by-step instructions for using the DVRescue graphic user interface to capture DV videotape formats. 
+<details markdown=1>
+  <summary markdown="span">Details</summary>
 <iframe class="embed-doc" src="https://docs.google.com/document/d/e/2PACX-1vRi-g_KLJyzX89ZISEYD9v9kH8c-MABKZxjq-mnC9zWFdcLA0FFgoTQU1C2VxMUhlGvvWUwYCxjT1p2/pub?embedded=true"></iframe>
+</details>
+
+## How to capture using the DVRescue CLI
+Commands and useage for capturing DV tapes using the dvrescue command line tools (CLI).
+<details markdown=1>
+  <summary markdown="span">Useage of capture commands</summary> <br />
+  
+**Basic useage:** <br />
+Below is the order you should type the commands for capturing using the CLI.
+
+1.) In the Terminal run ```dvrescue --list_devices``` to display the names of the devices connected to your computer. Note the name and device number of the one you want to use for capture. For example: <br />
+<img width="603" alt="Screenshot 2024-07-29 at 11 56 07 AM" src="https://github.com/user-attachments/assets/1ce94115-f6fc-467f-9b22-cb4e80763909">
+
+2.) You can either type ```dvrescue``` at the start of the command or include the path to the version of dvrescue you want to use. Example: ```/usr/local/bin/dvrescue```
+
+4.) If you only have one device connected, dvrescue will automatically select it (device 0). If you have more than one device connected, add the device number ```device://1``` (for the corresponding deck you want to use for capture). Example (using the deck listed in step 1): ```dvrescue device://0``` 
```

Comment:
hmmm, don't you have the full guid in your result such as
```
$ dvrescue --list_devices
90:a1838cb4:00000000: UltraStudio 4K Mini [DeckLink]
```

Reply:
Whoops, looks like that was an old example. Will update. So people need to add full number if they have more than one device plugged in right?

Reply:
@dericed See latest additions to the pull request for changes :)